From 2ad957d5da1fc91faf7c2c6562139d19f29f5b9c Mon Sep 17 00:00:00 2001
From: caro <caro@7cbeb6ba-43b4-40fd-8cce-4c39aea84d33>
Date: Fri, 22 Jun 2012 05:36:00 +0000
Subject: [PATCH] ecore_exe: fix compilation on fedora 18.

setpriority needs sys/resource.h. See:

http://linux.die.net/man/2/setpriority

I let sys/time.h, it does not hurt on linux and is anyway needed for BSD systems.


git-svn-id: svn+ssh://svn.enlightenment.org/var/svn/e/trunk@72684 7cbeb6ba-43b4-40fd-8cce-4c39aea84d33
---
 src/lib/ecore/ecore_exe.c |    7 ++-----
 2 files changed, 6 insertions(+), 5 deletions(-)

diff --git a/src/lib/ecore/ecore_exe.c b/src/lib/ecore/ecore_exe.c
index 7931a96..6db32a9 100644
--- a/src/lib/ecore/ecore_exe.c
+++ b/src/lib/ecore/ecore_exe.c
@@ -2,11 +2,6 @@
 # include <config.h>
 #endif
 
-#if defined (__FreeBSD__) || defined (__OpenBSD__) || defined (__NetBSD__)
-# include <sys/time.h>
-# include <sys/resource.h>
-#endif
-
 #include <stdlib.h>
 #include <stdio.h>
 #include <string.h>
@@ -14,6 +9,8 @@
 #include <sys/types.h>
 #include <unistd.h>
 #include <fcntl.h>
+#include <sys/time.h>
+#include <sys/resource.h>
 
 #ifdef HAVE_SYS_PRCTL_H
 # include <sys/prctl.h>
-- 
1.7.8.6

